src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -38,7 +38,7 @@
                     {% endif %}
                 {% endif %}
             </td>
-            <td></td>
+            <td>{{ form_widget(actionsForm.locationVisibility[location.id]) }}</td>
```

Comment:
You are right when you say that the visibility form item has nothing to do with `Location/Action`. But it also bothers me a bit that it has nothing to do with `Location/Visibility` :)

I'm still unsure wether client side form items such as this one should be known to the application layer or not. It would make sense if the location id form widgets were provided by the `Location\Visibility` form object:

```twig
{{ form_widget(visibilityForm.locationVisibility[location.id]) }}
```

The Form would know that it needs to generate a checkbox, or whatever the frontend needs, and the template wouldn't have to know about it.

An alternative is to consider that the frontend code's specific requirements are first and foremost configured in templates. But the fact is that for now, we have a good part of it in the FormTypes from the package.